migration/sql-files/029-add-space-id-wilt.sql

```diff
@@ -0,0 +1,6 @@
+-- Alter the tables
+ALTER TABLE work_item_link_types ADD space_id uuid;
```

Comment:
At this moment, it might be NULL. This is a question for @aslakknutsen, can we have `space_id` being NULL ?